types/chrome/chrome-cast/index.d.ts

```diff
@@ -766,13 +720,13 @@ declare namespace chrome.cast.media {
         currentTime: number;
         customData: Object;
         media: chrome.cast.media.MediaInfo;
+        playbackRate: number;
```

Comment:
Both of these need to be marked as optional, as per the docs specifying that they can be undefined.
```suggestion
        media?: chrome.cast.media.MediaInfo | undefined;
        playbackRate?: number | undefined;
```

Reply:
Unless I'm misreading, `media` must be non-null, as per the documentation. `playbackRate` on the other hand is nullable.

Reply:
whoops! sorry about that!